### PR TITLE
fix: fixes the issue with contrib-aws-resources not working

### LINKED ISSUE #3: Automatically configure the ECS Detector if the application is running on ECS
See https://github.com/lumigo-io/opentelemetry-js-distro#baseline-setup

### PR DESCRIPTION
**Pull request by AlmogBaku**
This change fixes the issue where the contrib's aws-resources didn't work or broke the agent.

closes #3

### CODE REVIEW

**Review by mmanciop — COMMENTED** [review]
@AlmogBaku how is this automatically tested?

**Comment by mmanciop** [comment]
> @GuyMoses, can you help with the test's issue:
> 
> > ❌ Building assets failed: Error: Building Assets Failed: Error: lumigo-java-distro-itests-dev: This CDK deployment requires bootstrap stack version '6', found an older version. Please run 'cdk bootstrap'.
> > [28](https://github.com/lumigo-io/opentelemetry-java-distro/actions/runs/4563146383/jobs/8051279373?pr=10#step:10:29)
> 
> I'm not sure if bootstraping will break things up for other users iצn this account (edited)

Bootstrap is remote: it runs a CloudFormation template in the account. If anything, it will unbreak anybody else that might be trying to use CDK in the region (`cdk bootstrap` is AWS-region dependent IIRC) :-)

**Comment by AlmogBaku** [comment]
tests are passing now

File: integration-tests/ecs/src/ecs-fargate-elb/ecs-fargate-elb.ts [hunk 1]
```diff
@@ -1,89 +1,115 @@
...
-                    cidrMask: 24,
-                },
-                {
-                    name: 'public-subnet',
-                    subnetType: SubnetType.PUBLIC,
-                    cidrMask: 24,
-                },
-            ],
-        });
+    const lumigoTokenSecret = EcsSecret.fromSecretsManager(
```
Comment (mmanciop) [inline integration-tests/ecs/src/ecs-fargate-elb/ecs-fargate-elb.ts:27]: We should use https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.SecretValue.html#static-plainwbrtextsecretspan-classapi-icon-api-icon-deprecated-titlethis-api-element-is-deprecated-its-use-is-not-recommended%EF%B8%8Fspan instead with the LumigoToken read from process.env

File: integration-tests/ecs/src/ecs-fargate-elb/ecs-fargate-elb.ts [hunk 2]
```diff
@@ -1,89 +1,115 @@
...
 
 export class EcsFargateElbStack extends Stack {
 
-    constructor(scope: Construct, id: string, props: StackProps) {
-        super(scope, id, props);
+  constructor(scope: Construct, id: string, props: StackProps) {
+    super(scope, id, props);
 
-        const lumigoTokenSecret = EcsSecret.fromSecretsManager(Secret.fromSecretNameV2(this, 'Secret', 'AccessKeys'), 'LumigoToken');
+    const suffix = props.tags?.['lumigo:suffix'] || '-dev';
```
Comment (mmanciop) [inline integration-tests/ecs/src/ecs-fargate-elb/ecs-fargate-elb.ts:25]: Replace the random suffix with the GA run id

**Comment by mmanciop** [comment]
Subsumed in #32

**Pull request closed without merging**